.circleci/config.yml

```diff
@@ -17,55 +17,43 @@ commands:
           command: npm test
 
 jobs:
```

Comment:
For the CI config, I've opted to change it to primarily using even-numbered releases, as they are the long-term-support versions, whereas the odd numbers are essentially development versions.  It doesn't really make sense to test against, say node 9 or 11, the majority of people use the stable versions such as 10 or 12.